### PR TITLE
Features/comments

### PR DESCRIPTION
**Pull request by gilemos**
put comments in the details view. I still want to make the cell size adjust according to the size of the comment, but for now it will cut the comment at the end of the 5th screen.
Screen Shot 2019-07-27 at 11 32 08 AM
Screen Shot 2019-07-27 at 11 32 19 AM

### CODE REVIEW

File: Travel Scheduler/Views/CommentsCell.m [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+//
+//  Created by gilemos on 7/27/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "CommentsCell.h"
+#import "Place.h"
+#import "UIImageView+AFNetworking.h"
+
+static UIImageView* makeImage(int width)
```
Comment (aliu917) [inline Travel Scheduler/Views/CommentsCell.m]: Hehe, I was making this mistake the whole time too, but the asterisk should be next to the method name like "static UIImageView *makeImage(int width)"

**Review by aliu917 — APPROVED** [review]
The comments look really good! I don't know if we necessarily have to make the table view cells adjust to all comment sizes because most apps don't do that but I do think it would be cool if shorter comments had a smaller size and maybe longer comments had a ... or +more button which would then elongate the comment (Idk but definitely stretch if we consider doing it at all)?

File: Travel Scheduler/Views/CommentsCell.m (hunk 1 above)
Comment (gilemos) [reply Travel Scheduler/Views/CommentsCell.m]: ooohh did not know! Gonna fix that, thanks!

**Comment by gilemos** [comment]
I loved the idea of the + button! It is definitely a stretch, but I would be happy if we had time to implement that

**Pull request merged**